addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -174,12 +182,12 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
 
     async _onSubmitBoothSelectionClick(ev) {
         ev.preventDefault();
-        let $form = this.$('.o_wbooth_registration_form');
-        let event_booth_ids = this.$('input[name=event_booth_ids]:checked').map(function () {
-            return parseInt($(this).val());
-        }).get();
+        const formEl = this.el.querySelector(".o_wbooth_registration_form");
+        const event_booth_ids = [
```

Comment:
`eventBoothIds` ?
While touching the line that uses it... feel free to also fix the method name.